src/core/public/notifications/toasts/global_toast_list.tsx

```diff
@@ -61,6 +61,7 @@ export class GlobalToastList extends React.Component<Props, State> {
   public render() {
     return (
       <EuiGlobalToastList
+        aria-label="Notification message list"
```

Comment:
I forgot to i18n this string, could you take care of it?

Reply:
addressed it